### PR TITLE
Improve converter

### PR DESCRIPTION
**Pull request by kostya13**
Add module revision to metadata
Create default types for configuration and rpc.
Add '_type' suffix to custom data types
Fix error with augments.
Add 'dsdl' output to yaml

### CODE REVIEW

**Comment by yoramw** [comment]
I suggest the derived from code will take the code snippet from an external resource so that we can easily have options in the future (might be xml_rpc, might be json_restconf)...
For now it is just xml_rpc, but it is good to plan ahead for additional plugins support

**Comment by kostya13** [comment]
This code don't works with xml_rpc. It works with DOM, prepared by pyang.

File: yttc/plugin/tosca.py [hunk 1]
```diff
@@ -74,53 +66,79 @@ def emit_yaml(ctx, modules, fd):
...
+        'data_types': {
+            'config': {
+                'properties': {
+                }},
+            'rpc': {
+                'properties': {
+                }}},
+        'node_types': {
+            main_module: {
+                'derived_from': 'cloudify.netconf.nodes.xml_rpc',
```
Comment (yoramw) [inline yttc/plugin/tosca.py]: This is what I referred to with the "xml_rpc"
It is better not to have this hardcoded inside the tosca.py, but load this from an external file which can be changed by specifying a different flag.

**Comment by kostya13** [comment]
I can add command line parameter, which specify base node_type.
By default it can be cloudify.netconf.nodes.xml_rpc. But if user want to configure output to another plugin, he can specify another type.

**Pull request merged**